csrc/reduction_utils.cuh

```diff
@@ -20,29 +20,47 @@
 #include "cuda_compat.h"
 
 namespace vllm {
+
```

Comment:
Hey @bnellnm  - I made some C++ interface changes to this file - Can you please take a look.

Reply:
looks ok to me.